express-mutations/server.ts

```diff
@@ -86,9 +88,11 @@ app.delete('/api/actors/:actorId', async (req, res, next) => {
     returning *
     `;
     if (actorId === undefined)
-      throw new ClientError(404, 'the actor id was not found');
+      throw new ClientError(400, 'the actor id was not found');
     const result = await db.query(sql, [actorId]);
     const deleted = result.rows[0];
+    if (deleted === undefined)
+      throw new ClientError(404, 'did not delete from database.');
```

Comment:
You could provide the same message here as you did above. ("could not find actor Id in database") That's because the only reason it won't delete is because the actor Id was not in the database.